AdlsSdkSamples/Program.cs

```diff
@@ -209,9 +163,12 @@ private static void RunFileTransfer(AdlsClient client)
                 stream.WriteLine("Hello I am the second line of upload.");
             }
             var destFile = remoteFileTransferPath + "/testremoteUploadFile.txt";
+            DataLakeFileClient file = client.GetFileClient(destFile);
             Console.WriteLine("Upload of the file:");
-            client.BulkUpload(fileName, destFile, 1); // Source and destination could also be directories
-            using (var readStream = new StreamReader(client.GetReadStream(destFile)))
+            file.Upload(fileName); // Source and destination could also be directories
+            Response<FileDownloadInfo> fileContents = file.Read();
+            Response<FileDownloadInfo> fileContentDown = file.Read();
```

Comment:
Why are we downloading the file twice?  Let's just `CopyTo` a `MemoryStream` if we want to use it multiple times.

Reply:
@tg-msft Updated.